patches/activity/001_drop_python2.patch

```diff
@@ -1101,7 +1102,7 @@ diff -r -c target/ya_activity/api/requestor_control_api.py src/ya_activity/api/r
               body=body_params,
               post_params=form_params,
               files=local_var_files,
-!             response_type='str',  # noqa: E501
+!             response_type='OneOfstringCreateActivityResult',  # noqa: E501
```

Comment:
This won't resolve to
```yaml
oneOf:
  - type: string
  - $ref: '#/components/schemas/CreateActivityResult'
```
Type lookup of `OneOfstringCreateActivityResult` will fail.


Please refer to https://github.com/golemfactory/ya-py-aioclient/pull/7/files#diff-de75c612a82d1078c3b1889f878dfda309dc055b47ea0f89145aa3683d47fd62